requirements.txt

```diff
@@ -0,0 +1,10 @@
+Click==7.0
+Flask==1.1.1
+Flask-SocketIO==4.2.1
+itsdangerous==1.1.0
+Jinja2==2.11.1
+MarkupSafe==1.1.1
+python-engineio==3.11.2
+python-socketio==4.4.0
+six==1.14.0
+Werkzeug==1.0.0
```

Comment:
AFAIK, Flask already contains Werkzeug, six, and other.

Reply:
*this file was autogenerated via venv and pip freeze command*